File: 1013-FibonacciNumber/1013-FibonacciNumber.py
# Last updated: 1/8/2026, 3:33:29 PM
class Solution(object):
    def fib(self, n):
        """
        :type n: int
        :rtype: int
        """
        a=0
        b=1
        for i in range(n+1):
            res=a
            t=a+b
            a=b
            b=t
        return res
        

File: 1013-FibonacciNumber/1013-FibonacciNumber.cpp
// Last updated: 1/8/2026, 3:33:32 PM
class Solution {
public:
    int fib(int n) {
        int a=0,b=1,t,res;
        for(int i=0;i<=n;i++){
            res=a;
            t=a+b;
            a=b;
            b=t;
        }
        return res;
    }
};